action.yml

```diff
@@ -53,5 +57,11 @@ runs:
           echo "IGNORE_DEV=true" >> $GITHUB_ENV
         fi
     - id: perform-source-composition-analysis
-      run: SEVERITY=${{ inputs.severity }} && ./contrast audit --file ${{ inputs.filePath }} --api-key ${{ inputs.apiKey }} --authorization ${{ inputs.authHeader }} --organization-id ${{ inputs.orgId }} --host ${{ inputs.apiUrl }} ${IGNORE_DEV:+"--ignore-dev"} ${FAIL:+"--fail"} ${SEVERITY:+"--severity"} ${SEVERITY:+"$SEVERITY"} --save
+      run: |
+        if [ "${{inputs.output_summary}}" = true ]
```

Comment:
There's no need to create an if statement and copy the command twice based on the value of this input, you can use shell parameter expansion instead, like the rest of the command uses. Can you please update this?

Reply:
I don't think `>> $GITHUB_STEP_SUMMARY` can be added as a shell expansion.  let me test on the internal copy.